api/ocs/__init__.py

```diff
@@ -519,7 +562,7 @@ def parse_observation(data: dict, num: int) -> Observation:
             if guide_group is not None:
                 for guide_data in guide_group[OcsProgramProvider._TargetEnvKeys.GUIDE_PROBE]:
                     guider = guide_data[OcsProgramProvider._TargetEnvKeys.GUIDE_PROBE_KEY]
-                    resource = Resource(guider, guider, None)
+                    resource = Resource(id=guider)
```

Comment:
I'm trying to phase out `name`, since `name` has always been the same as `id`. I don't think that we need both, and think that we could just use `description` if we want to give more detail. Three fields seems excessive.

I should probably reduce `__eq__` to just comparing `id`.